docs/querying/datasource.md

```diff
@@ -370,3 +370,55 @@ always be correct.
 - Join algorithms other than broadcast hash-joins.
 - Join condition on a column compared to a constant value.
 - Join conditions on a column containing a multi-value dimension.
+
+### `unnest`
+
```

Comment:
I forgot to add this to the tutorial and this section. Is there anything more we need to mention about it being only available for native queries @somu-imply ? 

> The `unnest` datasource is currently only available as part of a native query .

Reply:
I think it's fine. I am working on the SQL part of this and hoping it will be available soon